### ISSUE TITLE
Add to ImageHub

### ISSUE THREAD

**Issue by elimisteve** [coolya/image-app-tor#2]
I'd love to use this!  Can you please go through [these](https://community.scaleway.com/t/how-to-distribute-or-share-an-image/1129) steps so that this is included in the ImageHub for all to see and use?  Thanks!

**Comment by coolya** [comment]
Sorry for the huge delay. I will transfer it to scaleways community account if they accept it.

**Comment by moul** [comment]
yep, transfer me the ownership, then I prepare everything in scaleway-community and add you as collaborator

**Comment by coolya** [comment]
@moul I guess you have to delete your fork otherwise I can't transfer it.

**Comment by moul** [comment]
done :)

**Comment by moul** [comment]
@coolya, transfer complete, you've been invited in the organization, I will do some fixes on the image to made the image multiarch-ready, I will create a PR so you can comment and merge by yourself

**Comment by coolya** [comment]
@moul thank for the fast response :) I fixed a minor bug in the pull request.
